### PR TITLE
Adding HappyPath testing scenario for AI Studio

### PR DESCRIPTION
**Pull request by ScrewTSW**

### CODE REVIEW

**Review by odockal — APPROVED** [review]

File: docs/scenarios/aistudio/aistudio-testing.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+2. Install a new extension from OCI Image -> `ghcr.io/projectatomic/ai-studio:latest` (possible to also test `:nightly` for latest builds - every commit from `https://github.com/projectatomic/ai-studio`)
+3. Verify that AI Studio extension is present under Settings -> Extensions
+4. Verify that Studio extension is present in the left navbar
+
+## Running an AI Studio demo
+
+1. Click on Studio extension in the left navbar
+2. Click on Recipe Catalog
+3. Click on ChatBot demo application
+4. Install demo application - will download a rather large LLM model from HuggingFace
```
Comment (odockal) [inline docs/scenarios/aistudio/aistudio-testing.md]: Start AI app is the option, right? There are two models that can be picked. I would also suggest to switch the tab and verify the default model is picked. we can be also specific about the model to use.

Comment (odockal) [reply docs/scenarios/aistudio/aistudio-testing.md]: I mean, both models are from huggingFace.

Comment (ScrewTSW) [reply docs/scenarios/aistudio/aistudio-testing.md]: We will have to switch over to an open source model downloaded locally once the update with the feature is provided. model TBD

**Review by odockal — CHANGES_REQUESTED** [review]
Please address the changes.

File: docs/scenarios/aistudio/aistudio-testing.md (hunk 1 above)
Comment (ScrewTSW) [reply docs/scenarios/aistudio/aistudio-testing.md]: fixed

**Review by odockal — APPROVED** [review]

**Pull request merged**